### PR TITLE
Update katello-host-tools spec python macros

### PR DESCRIPTION
**Pull request by pcreech**
This will allow us to be more compliant with current python packaging
standards

### CODE REVIEW

File: packages/client/katello-host-tools/katello-host-tools.spec [hunk 1]
```diff
@@ -3,8 +3,6 @@
 %global zypper_install (0%{?suse_version} > 0)
 %global build_tracer 0%{?rhel} >= 7 || 0%{?fedora} || 0%{?suse_version}
 
-%{!?python_sitelib: %global python_sitelib %(%{__python} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")}
```
Comment (pcreech) [inline packages/client/katello-host-tools/katello-host-tools.spec:6]: New distro versions will fail to build whenever it evaluates a `__python` macro

**Review by zjhuntin — APPROVED** [review]
ACK

**Comment by evgeni** [comment]
it's hard to verify the el6 stuff still works, given we don't build el6 anymore…

**Comment by pcreech** [comment]
el6 build on the unused, but still existent foreman-client-nightly-el6 tag:  http://koji.katello.org/koji/taskinfo?taskID=482456

**Pull request merged**